GitHubTeams.ps1

```diff
@@ -276,6 +315,488 @@ filter Get-GitHubTeamMember
     return (Invoke-GHRestMethodMultipleResult @params | Add-GitHubUserAdditionalProperties)
 }
 
+function New-GitHubTeam
+{
+<#
+    .SYNOPSIS
+        Creates a team within an organization on GitHub.
+
+    .DESCRIPTION
+        Creates a team within an organization on GitHub.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OrganizationName
+        The name of the organization to create the team in.
+
+    .PARAMETER TeamName
+        The name of the team.
+
+    .PARAMETER Description
+        The description for the team.
+
+    .PARAMETER MaintainerName
+        A list of GitHub IDs for organization members who will become team maintainers.
+
+    .PARAMETER RepositoryFullName
```

Comment:
As I think about this more, we've already specified the `OrganizationName` in the parameter list.  Is there a specific reason why we should make users specify the _full_ repositoryName here as opposed to just `RepositoryName` and then _build_ the full name internally?